src/styles/layout/_header.scss

```
@@ -0,0 +1,288 @@
+      @include typography($BoldMD, $White);
+    }
+  }
+
+  &__left {
+    li {
+      padding: 0 8px;
+      transition: box-shadow 0.3s ease-in-out;
+      a {
+        @include typography($BoldMD, $White);
+      }
+    }
+    > li:hover,
+    > li:focus,
+    > li:active {
+      transition: box-shadow 0.3s ease-in-out;
+      box-shadow: 0 0 0 1px $White inset;
+    }
+  }
+
+  &__right {
+    transition: box-shadow 0.3s ease-in-out;
+    a {
+      display: block;
+      height: 32px;
+      padding: 0 12px;
+      @include typography($BoldMD, $White);
+    }
+    &:hover,
+    &:focus,
+    &:active {
+      transition: box-shadow 0.3s ease-in-out;
+      box-shadow: 0 0 0 1px $White inset;
+    }
+  }
+}
+
+.header-layer {
+  width: 100%;
+  position: absolute;
+
+  &__popup--address,
+  &__popup--login,
+  &__popup--login-extend {
+    @include layer;
+  }
+
+  &__popup--address {
+    position: absolute;
+    z-index: 50;
+    top: 0;
+    left: 131px;
+    width: 320px;
+    height: 126px;
+    @include popupAnimation;
+
+    .shape-triangle {
+      @include clip-path-triangle(10px);
+    }
+    .layer-content {
+      height: 100%;
+      display: flex;
+      flex-direction: column;
+      .btn {
+        display: flex;
+        gap: 8px;
+        margin-left: auto;
+        margin-top: auto;
+        text-align: right;
+
+        .btn--continue {
+          @include button();
+        }
+        .btn--chage-address {
+          @include button();
+        }
+      }
+    }
+  }
+
+  &__popup--login {
+    width: 235px;
+    position: absolute;
+    z-index: 50;
+    top: 0;
+    right: 172px;
+    @include popupAnimation;
+    .shape-triangle {
+      @include clip-path-triangle(188px);
+    }
+    .btn-login {
+      @include button();
+      width: 160px;
+    }
+    div {
+      @include typography($BodySM);
+      a {
+        color: $SecondaryNavy;
+      }
+    }
+  }
+
+  &__popup--login-extend {
+    width: 335px;
+    position: absolute;
+    z-index: 50;
+    top: 0;
+    right: 126px;
+    @include popupAnimation;
+    .shape-triangle {
+      @include clip-path-triangle(242px);
+    }
+
+    .top-area {
+      width: 100%;
+      text-align: center;
+      padding-bottom: 8px;
+      border-bottom: 1px solid $Gray100;
+
+      .btn-login {
+        @include button();
+        width: 160px;
+        margin: 0 auto 8px;
+      }
+      a {
+        color: $SecondaryNavy;
+      }
+    }
+
+    .bottom-area {
+      display: flex;
+      text-align: left;
+
+      .left-list,
+      .right-list {
+        width: 153px;
+        div {
+          @include typography($BoldMD);
+          padding-bottom: 4px;
+        }
+
+        ul {
+          display: flex;
+          flex-direction: column;
+          gap: 4px;
+        }
+      }
+    }
+  }
+
+  &__popup--address.hidden {
+    display: none;
+  }
+  &__popup--login-extend.hidden {
+    display: none;
+  }
+  &__popup--login.hidden {
+    display: none;
+  }
```

이 부분은 3번 정의하기보다는 `hidden` 이라는 클래스만 따로 정의해두어도 되겠다는 생각이 드네요!
hidden css는 프로젝트 전반에 쓰일 수 있을 테니 global ? 혹은 main ?에 정의해두면 될 것 같구요

```suggestion
.hidden {
  display: none;
}
```

---

모달 부분을 제일 마지막으로 만들면서 일일이 하나씩 따로 하면서 확인했었습니다.
급하게 PR을 올리느라 생각을 못했네요!! 수정하겠습니다 😃